src/libecalc/fixtures/cases/minimal/minimal_yaml.py

```diff
@@ -37,3 +37,34 @@ def minimal_model_yaml(fuel_rate: Union[int, str] = 50):
         )
 
     return minimal_model_yaml
+
+
+@pytest.fixture
+def minimal_model_yaml_factory_2():
+    def minimal_model_yaml(fuel_rate: Union[int, str] = 50):
+        return YamlModel(
+            name="minimal_model_2",
+            source=f"""
+FUEL_TYPES:
+  - NAME: fuel_2
+    EMISSIONS:
+      - NAME: co2
+        FACTOR: 2
+
+START: 2020-01-01
+END: 2023-01-01
+
+INSTALLATIONS:
+  - NAME: minimal_installation_2
+    HCEXPORT: 0
+    FUEL: fuel
```

Comment:
invalid model, refer to `fuel`but has `fuel_2`. should be valid imo

Reply:
good point, it somehow seemed to pass the validation when I ran it 🤷‍♂️